app/mailers/client_mailer.rb

```diff
@@ -0,0 +1,9 @@
+# class to send mails to client
+class ClientMailer < ApplicationMailer
+  default from: 'taxi@taxi.com'
+
+  def welcome_email(order_id)
+    @driver = Driver.find(Order.find_by(id: order_id).driver_id).name
+    mail(to: 'email@email.ua', subject: 'Taxi')
```

Comment:
But it is sent to email@email.ua. Also I tested with mailcatcher, all seems working